src/Inciter/Partitioner.h

```diff
@@ -387,6 +388,8 @@ class Partitioner : public CBase_Partitioner< HostProxy,
     //! \param[in] p The PE uniquely assigns the node IDs marked listed in ch
     //! \param[in] cn Vector containing the set of potentially multiple chare
     //!   IDs that we own (i.e., contribute to) for all of our node IDs.
+    //! \param[in] ce Map associating a vector of chare IDs to edges (at which
```

Comment:
`ce` is a map that associates a vector of chare IDs to edges. That is, the map key is an edge (an array of two integers, node IDs of mesh vertices, see UnsMesh::Edge) and the map value is a vector of chare IDs. This map is empty if no initial mesh refinement happened.